packages/adapters-library/src/adapters/aave-v3/products/rewards/aaveV3RewardsAdapter.ts

```diff
@@ -132,93 +131,43 @@ export class AaveV3RewardsAdapter implements IProtocolAdapter {
     ]
   }
 
-  private detectPositionEventSignature(
-    eventSignature = 'Transfer(address,address,uint256)',
-  ): string {
-    return ethers.id(eventSignature)
-  }
-
-  /**
-   * Checks if the user has ever opened a position in AaveV3
-   * Return AToken addresses if found
-   */
-  private async openPositions(userAddress: string): Promise<string[]> {
-    const topic0 = this.detectPositionEventSignature()
-    const topic1 = ethers.zeroPadValue(ZERO_ADDRESS, 32)
-    const topic2 = ethers.zeroPadValue(userAddress, 32)
-
-    const logs = await this.provider.getLogs({
-      topics: [topic0, topic1, topic2],
-      fromBlock: '0x',
-      toBlock: 'latest',
-    })
```

Comment:
If we have to use getLogs here it defeats the purpose. No other adapter is doing this internally.